cumulusci/core/flowrunner.py

```diff
@@ -115,7 +115,7 @@ def for_display(self):
         skip = ""
         if self.skip:
             skip = " [SKIP]"
-        return "{step_num:8}: {task_name}{skip}".format(
+        return "{step_num}: {task_name}{skip}".format(
```

Comment:
that was to add some whitespace and keep the task names aligned... does it look weird?

Reply:
It wasn't a valid format and I didn't feel like tracking down what it's supposed to be. Feel free to fix it as long as it doesn't break the tests.

Reply:
oh i just checked in ipython and we just need to str(self.step_num)